src/snowcli/cli/core/stage.py

```diff
@@ -49,7 +49,7 @@ def stage_get(
     environment: str = EnvironmentOption,
     name: str = typer.Argument(..., help="Stage name"),
     path: Path = typer.Argument(
-        Path("."),
+        Path.cwd(),
```

Comment:
Should we add basic test for this method? Probably assuring this path is same before and after refactor will be enough. WDYT?